content/influxdb/v2.0/upgrade/v1-to-v2/migrate-cqs.md

```diff
@@ -80,14 +86,210 @@ from(bucket: "my-db/")
   )
 ```
 
-### Continuous queries with a RESAMPLE clause
+### Convert specific InfluxQL CQ statements and clauses to Flux
```

Comment:
```suggestion
### Convert InfluxQL continuous queries (CQ) to Flux
```

Reply:
Review the following statements and clauses to see how to convert your continuous queries (CQ) to Flux: